demos/paging.html

```diff
@@ -76,7 +76,8 @@
             ],
             columnMode: 'force',
             paging: {
-              mode: 'internal'
+              mode: 'internal',
+              size: 12
```

Comment:
Went to 12 because the data source is 100 rows and I wanted to verify that everything looks good with a row count not evenly divisible by the page size.